parcels/particle.py

```diff
@@ -198,15 +201,6 @@ def __repr__(self):
     def delete(self):
         self.state = ErrorCode.Delete
 
-    def succeeded(self):
-        self.state = ErrorCode.Success
-
-    def isComputed(self):
-        return self.state == ErrorCode.Success
-
-    def reset_state(self):
-        self.state = ErrorCode.Evaluate
-
```

Comment:
Again, depending on your view of access protection for the `state` variable, we either leave it out as such, or we also introduce those functions in the ParticleSet. Choice is up to you.